app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,93 @@
+require './config/environment'
+
+class TweetController < ApplicationController
+
+  get '/tweets' do
+    if session[:user_id]
+      @user = User.find_by(session[:user_id])
+      @tweets = Tweet.all
+      #binding.pry
+      erb :tweets
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if logged_in?
+      erb :new
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+      if logged_in?
+        if params[:content] == ""
+          redirect to "/tweets/new"
+        else
+          @tweet = current_user.tweets.build(content: params[:content])
+          if @tweet.save
+            redirect to "/tweets/#{@tweet.id}"
+          else
+            redirect to "/tweets/new"
+          end
+        end
+      else
+        redirect to '/login'
+      end
+    end
+
+  get '/tweets/:id' do
+    if logged_in?
+      @tweet = Tweet.find_by(params[:id])
+      erb :show
+    else
+      redirect to '/login'
+    end
+  end
+
+  get '/tweets/:id/edit' do
+    #binding.pry
+    if logged_in?
+      @tweet = Tweet.find_by(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?